Makefile

```diff
@@ -1,6 +1,10 @@
 # Copyright 2016, Pablo Ridolfi
```

Comment:
put your copyright here like this:
Copyright 2016, Iván Castellucci Vidal <ivanc.vidal@gmail.com>